app/utils/answersHelpers/ConsignmentAnswersHelper.scala

```diff
@@ -133,6 +144,41 @@ class ConsignmentAnswersHelper(userAnswers: UserAnswers)(implicit messages: Mess
     call = Some(Call(GET, "#"))
   )
 
+  def inlandModeOfTransportRow: Option[SummaryListRow] = getAnswerAndBuildRow[InlandMode](
+    page = InlandModeOfTransportPage,
+    formatAnswer = formatAsText,
+    prefix = "unloadingFindings.inlandModeOfTransport.label",
+    id = None,
+    call = None
+  )
+
+  def countryOfDestinationRow: Option[SummaryListRow] = getAnswerAndBuildRow[Country](
+    page = CountryOfDestinationPage,
+    formatAnswer = formatAsText,
+    prefix = "unloadingFindings.countryOfDestination",
+    id = None,
+    call = None
+  )
+
+  def containerIndicatorRow: Option[SummaryListRow] =
+    userAnswers.ie043Data.Consignment
+      .map {
+        consignment =>
+          consignment.containerIndicator match {
+            case Number0 => false
+            case Number1 => true
+          }
+      }
```

Comment:
Think you can remove this now right? Since you added a `formatAsYesOrNo` method that accepts a `Flag` and does the boolean conversion for you

Reply:
I think it's required as later we need containerIndicator as boolean where we use this buildRow       
.map {
        containerIndicator =>
          buildRow(
            prefix = "unloadingFindings.containerIndicator",
            answer = formatAsYesOrNo(containerIndicator),
            id = None,
            call = None
          )
      }